projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.html

```diff
@@ -1,36 +1,36 @@
-<div id="govuk-accordion__section-{{ sharedCase.caseId }}" class="govuk-accordion__section">
+<div [id]="buildElementId('govuk-accordion__section')" class="govuk-accordion__section">
```

Comment:
Does `buildElementId(...)` return different values at different times? If not, can this be done with interpolation rather than a binding as it'll be quicker:

`id="{{ buildElementId('govuk-accordion__section') }}"`

The same for all the other `[id]` bindings too...